src/components/OSCALLoader.js

```diff
@@ -99,8 +99,30 @@ function restPatch(
     );
 }
 
+class ErrorBoundary extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { error: undefined };
+  }
+
+  static getDerivedStateFromError(error) {
+    return { error };
+  }
+
+  componentDidCatch(error, errorInfo) {
+    this.setState({ error });
+    console.error({ ...this.state, errorInfo });
```

Comment:
We probably also want to remove this

Reply:
Yeah I was wondering why that was removed

Reply:
Actually it doesn't work without `componentDidCatch`,